### ISSUE TITLE
create_many doesn't return Array of Entity

### ISSUE THREAD

**Issue by famano** [hanami/hanami#856]
hi, I implement create_many method in my repository like the following code.
```ruby
class SomeRepository < Hanami::Repository
  def create_many(list)
    command(
      :create,
      somes,
      use: [:timestamps],
      result: :many
    ).call(list)
  end
end
```
Then I call `SomeRepository.new.create_many([{ attr: "something" }, { attr: "something" }])`, It doesn't returns Array of Entity of "Some", but Array of `ROM::Struct::Some`.
According to the following guide, create_many should return Array of Entity. How can I fix this problem?
http://hanamirb.org/guides/1.0/models/repositories/#saving-more-than-one-record

**Comment by jodosha** [comment]
@famano I manually tried it and it looks like a bug.

You can temporary fix this problem just appending `.map {|e| Some.new(e) }`.

**Comment by AlfonsoUceda** [comment]
@famano sorry for the problem, can you tell us if that workaround worked?

**Comment by mereghost** [comment]
Working on it.

**Comment by famano** [comment]
@jodosha @AlfonsoUceda That works well. Thank you for finding fixation.

**Comment by jodosha** [comment]
We want to transform all the CRUD methods of repositories and all the associations methods to work in bulk mode.

Eg.

```ruby
repository.create(title: "Hello Ruby")
  # => Book

repository.create([{ title: "Hello Ruby" }, { title: "Hello Hanami" }])
  # => [Book, Book]
```

@mereghost is working on this.

**Comment by jodosha** [comment]
As we merged hanami/model#471, bulk operations can be implemented like this:

```ruby
class UserRepository < Hanami::Repository
  def create_many(data)
    command(create: :users, result: :many).call(data)
  end
end
```
